generate-sample-cert.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/sh
 openssl genrsa 1024 > sample.key
-openssl req -new -x509 -nodes -sha1 -days 3650 -key sample.key > sample.cert
-cat sample.cert sample.key > sample.pem
-rm sample.key sample.cert
-
+openssl req -new -x509 -nodes -sha1 -days 3650 -key sample.key > sample.crt
+cat sample.crt sample.key > sample.pem
```

Comment:
BTW is the .pem file still needed?  I see you're configuring it to use .key and .crt separately.